docs/tr/week08/08-1.md

```diff
@@ -0,0 +1,204 @@
+
+## [Arıtan Otokodlayıcılar](https://www.youtube.com/watch?v=ZaVP2SY23nc&t=1384s)
+
+<!-- ## [Denoising Autoencoders](https://www.youtube.com/watch?v=ZaVP2SY23nc&t=1384s)-->
+
+[7. haftanin uygulamasinda](https://atcold.github.io/pytorch-Deep-Learning/en/week07/07-3/), arıtan otokodlayıcıdan bahsetmiştik.
+Bu yapı, veri gösterimini bozuk girdiyi orjinal veriye geri dönüştürmeye çalışarak öğrenir. 
+Detaylıca bahsedersek, sistemi , bozulmuş veriler veri manifoldundan uzaklaştıkça karesel bir oranda büyüyen bir enerji fonksiyonu üretebilmesi için eğitiyoruz.
+
+<!-- In [week 7's practicum](https://atcold.github.io/pytorch-Deep-Learning/en/week07/07-3/), we discussed denoising autoencoder. The model tends to learn the representation of the data by reconstructing corrupted input to the original input. More specifically, we train the system to produce an energy function that grows quadratically as the corrupted data move away from the data manifold. -->
+
+
+<center>
+<img src="{{site.baseurl}}/images/week08/08-1/fig6.png" height="75%" width="75%"/><br>
+<b>Fig. 5</b>:Arıtan Otokodlayıcı Mimarisi
+</center>
+
+### Problemler
+
+<!-- ### Issues -->
+
+Ancak arıtan otokodlayıcılar ile ilgili çeşitli sorunlar bulunmaktadır. 
+Öncelikle, cok boyutlu ve sürekli bir uzayda, veriyi bozmanın sonsuz sayıda yolu vardır. Bu yüzden enerji fonksiyonunu bir sürü değişik bölgede yukarı itmenin de garantisi yokturç
+Bir diger problem ise saklı degişkenlerin noksanlığından ötürü arıtan otokodlayicilar göruntulerde kötu performans sergilemektedir.
+Bir resmi yeniden oluşturmanın sayısız yöntemi olduğundan, sistem çeşitli tahminler üretir ve bilhassa iyi öznitelikler öğrenemez. 
+Ayrıca, manifoldun ortasındak kalan bozunmuş noktalar iki taraftan da onarmamız mümkün. Bu durum enerji fonksiyonumuzda düz bölgeler oluşmasına neden olur ve genel performansı etkiler.
+
+<!-- However, there are several problems with denoising autoencoders. One problem is that in a high dimensional continuous space, there are uncountable ways to corrupt a piece of data. So there is no guarantee that we can shape the energy function by simply pushing up on lots of different locations. Another problem with the model is that it performs poorly when dealing with images due to the lack of latent variables. Since there are many ways to reconstruct the images, the system produces various predictions and doesn't learn particularly good features. Besides, corrupted points in the middle of the manifold could be reconstructed to both sides. This will create flat spots in the energy function and affect the overall performance. -->
+
+## Diger Karşıtsal Yontemler 
+
+
+Karşıtsal ıraksama(Contrastive divergence), Oran Eşleştirme(Ratio Matching) , Gürültüye Karşıt Kestirim(Noise Contrastive Estimation), ve Minimum OLasılık Akışı(Minimum Probability Flow) gibi çeşitli karşıtsal yöntemler lıteratürde bulunmaktadır. Bu bölümde, karşıtsal ıraksamanın arkasında yatan temel fikirden bahsedeceğiz. 
```

Comment:
```suggestion
Karşıtsal ıraksama (Contrastive divergence), Oran Eşleştirme (Ratio Matching), Gürültüye Karşıt Kestirim (Noise Contrastive Estimation) ve Minimum Olasılık Akışı (Minimum Probability Flow) gibi çeşitli karşıtsal yöntemler l
literatürde bulunmaktadır. Bu bölümde, karşıtsal ıraksamanın arkasında yatan temel fikirden bahsedeceğiz. 
```

Reply:
There's still a typo here. ' yöntemler l' . Resolving for other typos it but please fix it for next PR.

Reply:
fixed